Source/WebCore/Modules/identity/DigitalCredential.idl

```diff
@@ -30,4 +30,5 @@
     Conditional=WEB_AUTHN,
 ] interface DigitalCredential : BasicCredential {
     [SameObject] readonly attribute ArrayBuffer data;
```

Comment:
Aside: should this be a Uint8Array instead? See https://github.com/w3ctag/design-principles/issues/463.

Reply:
It’s probably going to change to an object, but otherwise yes: will change it to a Uint8Array.

Reply:
Filed an issue on the spec so it won’t be forgotten